app/code/Magento/Customer/Block/CustomerScopeData.php

```diff
@@ -50,4 +50,31 @@ public function getWebsiteId()
     {
         return (int)$this->_storeManager->getStore()->getWebsiteId();
     }
+
+    /**
+     * Get the invalidation rules json encoded
+     *
+     * @return bool|string
+     * @throws \InvalidArgumentException
+     */
+    public function getSerializedInvalidationRules()
+    {
+        return $this->serializer->serialize(
```

Comment:
I don't think serialization in block is a good idea. It is better to just return raw data and JSON (or whatever else) encode in template.

Due to such `get...Json` methods in Magento 1 some extensions had to decode string and then re-encode just to modify array.